### DISCUSSION TITLE
Postgresql Support

### DISCUSSION THREAD

**Discussion by richardsondx** [customink/lamby#142] [Ideas]
It would be great to add support for postgresql:
1. The current documentation provide support for mysql but not postgresql
2. The .devcontainer Dockerfile has instruction for mysql but there's not version for a postgresql installation.

**Comment by metaskills** [comment]
There are docs on how this is not in scope for the Lamby gem. It could be an option for the cookiecutter. So moved that issue to here. https://github.com/customink/lamby-cookiecutter/issues/27

**Reply by metaskills** [reply]
Till then tho, do you see how you can change out the docker compose file and gems on your own?
